src/uct/ib/rc/accel/rc_mlx5_iface.c

```diff
@@ -356,6 +346,33 @@ static ucs_status_t uct_rc_mlx5_iface_tag_recv_cancel(uct_iface_h tl_iface,
 }
 #endif
 
+static ucs_status_t
+uct_rc_mlx5_iface_parse_srq_topo(uct_ib_mlx5_md_t *md,
+                                 uct_rc_mlx5_iface_common_config_t *config,
+                                 uct_rc_mlx5_srq_topo_t *topo_p)
+
+{
+    int i;
+
+    for (i = 0; i < config->srq_topo.count; ++i) {
+        if (!strcasecmp(config->srq_topo.types[i], "list")) {
+            *topo_p = UCT_RC_MLX5_SRQ_TOPO_LIST;
+            return UCS_OK;
+        } else if (!strcasecmp(config->srq_topo.types[i], "cyclic")) {
+            if (!(md->flags & UCT_IB_MLX5_MD_FLAG_DEVX_RC_SRQ)) {
+                continue;
```

Comment:
comment: real cyclic list requires DevX support
BTW, should we check some HCA capability flag here?

Reply:
> BTW, should we check some HCA capability flag here?

@yosefe if I'm not mistaken, there is no such capability flag. in #4831 just checking only RMP capability flag
I see that there is only `ib_cyclic_striding_wq` which indicates that LIST and CYCLIC MP RQs are supported